docs/maintain/maintain-guides-how-to-validate-polkadot.md

```diff
@@ -642,13 +640,79 @@ other peers over the network.
 ./target/release/polkadot purge-chain
 ```
 
-## VPS List
+:::info
+
+Check out the [Substrate StackExchange](https://substrate.stackexchange.com/) to quickly get the answers you need.
+
+:::
+
+## Note about VPS
+
+VPS providers are very popular for running servers of any kind.
+Extensive benchmarking was conducted to ensure that VPS servers are able to keep up with the work load in general.
+This does not mean that every provider who advertises a specific performance will also deliver it.
+Please do your own verification that you get the advertised performance before running a live validator on it.
+The following server types showed acceptable performance. Please note that this is not an endorsement in any way:
+- GCP's *c2* and *c2d* machine families
+- AWS's *c6id* machine family
+
+The following additional configurations were applied to the instances to tune their performance:
+
+### Disable [SMT](https://en.wikipedia.org/wiki/Simultaneous_multithreading)
+As critical path of Substrate is single-threaded we need to optimize for single-core CPU performance. The node still profits from multiple cores when doing networking and other non-runtime operations. It is therefore still necessary to run it on at least the minimum required number of cores.
+Disabling SMT improves the performance as each vCPU becomes mapped to a physical CPU core rather than being presented to the OS as two logical cores.
+SMT implementation on Intel is called Hyper-Threading, on AMD Zen it is 2-way SMT. To disable SMT in runtime:
+```bash
+for cpunum in $(cat /sys/devices/system/cpu/cpu*/topology/thread_siblings_list | cut -s -d, -f2- | tr ',' '\n' | sort -un)
+do
+  echo 0 > /sys/devices/system/cpu/cpu$cpunum/online
+done
+```
+It will disable every other (vCPU) core.
+
+To save changes permanently add `nosmt=force` as kernel parameter. Edit `/etc/default/grub` and add `nosmt=force` to `GRUB_CMDLINE_LINUX_DEFAULT` variable and run `sudo update-grub`. After the reboot you should see half of the cores are offline. Run `lscpu --extended` to confirm.
+
+### Disable automatic NUMA balancing
+If you have multiple physical CPUs (CPU0 and CPU1) in the system each with its own memory bank (MB0 and MB1) than it is usually slower for a CPU0 to access MB1 due to the slower interconnect. To prevent the OS from automatically moving the running Substrate process from one CPU to another thus causing an increased latency, it is recommended to disable automatic NUMA balancing.
```

Comment:
```suggestion
If you have multiple physical CPUs (CPU0 and CPU1) in the system each with its own memory bank (MB0 and MB1), then it is usually slower for a CPU0 to access MB1 due to the slower interconnection. To prevent the OS from automatically moving the running Substrate process from one CPU to another and thus causing an increased latency, it is recommended to disable automatic NUMA balancing.
```